vm/usb/Makefile

```diff
@@ -14,7 +14,7 @@ SCRIPTS = ../../../scripts
 
 HOST_BUILD_FILES = \
 	build/host/data/appvm-usb/blk/root.img \
-	build/host/data/appvm-usb/Image\
+	build/host/data/appvm-usb/vmlinux \
```

Comment:
This changes vm/usb to x86_64 from aarch64 only. For testing only?

Reply:
This is a proper fix for old hack. We use generic kernel for VMs, not the host one anymore.

Reply:
I mean - won't aarch64 still require `Image` for guest kernels? Or am I mistaken?

Reply:
This hack is probably from the times when we used to use imx8 host kernel for VMs. All VMs have been using generic kernel for a long time now.